### PR TITLE
Removes "+" --> "plus" wordfilter on smartfridge

### PR DESCRIPTION
**Pull request by 9600bauds**
I'll name my pills "Arith+Antitox" if I want to, goddamit
This seems to have been some sort of sanitization to prevent HTML from popping up on VV but the + sign alone can't do anything I don't think, tested using VV on a pill with + in it's name, it's all good

### CODE REVIEW

**Comment by PJB3005** [comment]
Just remove that entirely. People shouldn't be allowed to make pills with unsanitized HTML in the first place.

**Comment by 9600bauds** [comment]
The chemmaster has reject_bad_text() on the naming so sure
Renaming food is also sanitized

File: code/game/machinery/kitchen/smartfridge.dm [hunk 1]
```diff
@@ -229,7 +229,7 @@
 			if(!user.drop_item(O, src))
 				return 1
 
-			var/sanitized_name = sanitize(O.name, list("\"" = "", "'" = "", "+" = "plus", ";" = "", "^" = "", "&" = "", "<" = "", ">" = ""))
+			var/sanitized_name = sanitize(O.name)
```
Comment (9600bauds) [inline code/game/machinery/kitchen/smartfridge.dm]: Do I leave in this sanitize() call?

Comment (ComicIronic) [inline code/game/machinery/kitchen/smartfridge.dm]: No, it will just confuse people in the future.

**Comment by nuklearcellphoneg** [comment]
Hi, does this fix not being able to do for example Keloderma 5/5 or Keloderma 10/10

**Comment by 9600bauds** [comment]
Probably

**Comment by nuklearcellphoneg** [comment]
God I hope so

**Pull request merged**